homeassistant/components/zamg/sensor.py

```diff
@@ -190,6 +191,7 @@ class ZamgSensorEntityDescription(SensorEntityDescription, ZamgRequiredKeysMixin
             cv.ensure_list, [vol.In(SENSOR_KEYS)]
         ),
         vol.Optional(CONF_STATION_ID): cv.string,
+        vol.Optional(CONF_UNIQUE_ID): cv.string,
```

Comment:
We no longer accept additions or changes to the yaml config of integrations. See: https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md

This integration needs to be refactored to use a config flow and config entries.

https://developers.home-assistant.io/docs/config_entries_config_flow_handler

Also, we probably won't allow a config option to set a unique id. The unique_id should be set by the integration by using some appropriate data:
https://developers.home-assistant.io/docs/entity_registry_index#unique-id-requirements

Reply:
Ok, i will try to update the code to use a config flow